### ISSUE TITLE
Grab Campaign Contribution Data

### ISSUE THREAD

**Issue by JohnCRuf** [JohnCRuf/alderman_machine#39]
After addressing #38, if I find a positive effect, it could be due to an intertemporal tradeoff between current election odds and next year. The basic theory would be that giving to current supporters induces campaign contributions, of which individual contributions matter most. Thus aldermen face a tradeoff -- they can give to their supporters to increase their war chest and roll over the war chest to the next election, or they can target the median voter now to secure this year's win. 

Thus, this would rationalize the (hypothetical) finding that swapping secure aldermen for new, less secure aldermen finds an effect but not swapping out aldermen in competitive elections. You only care about tomorrow, given that you're confident you'll win today.

**Comment by JohnCRuf** [comment]
So I finally got around to this, as I thought it may increase the amount of wards I could use for #38 -- as many of the entrenched incumbents I'd like to use do not have any challengers. I hoped that perhaps I could use the "top 5" donating precincts instead of the top 5 precincts who supported them in a previous election.

An example from Alderman Burke is given below: From 2012-2022 the 27 precinct gave Alderman Burke over $100,000 in individual contributions -- largely in in $2.5-5,000 increments. There's something really screwed up with this data and its highly unlikely to be the way I processed it.  Furthermore I'm not even sure how these increments are legal -- the individual contribution limit is $250 for individuals and $1500 for businesses that work for the city. Is there no limit on the contributions of businesses who do not work for the city?
 
![image](https://github.com/JohnCRuf/alderman_machine/assets/96131398/b61661f3-6d1b-45b9-b7ce-e005b1f3539c)
